catalog/spatial/geocoding/spatial-geocoding-plugin/src/test/groovy/org/codice/ddf/spatial/geocoding/plugin/GeoCoderPluginTest.groovy

```diff
@@ -212,6 +212,20 @@ class GeoCoderPluginTest extends Specification {
         geoCoderPlugin.getRadius() == 15
     }
 
+    def 'test update with null configuration'() {
+        setup:
+        def map = new HashMap<String, Object>()
+        map.put("radiusInKm", 15)
```

Comment:
LGTM. If we want we could change the `radiusInKm` to public in the `GeoCoderPlugin` and reference it here.